pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 16.0.0
+version: 1.0.0+30
```

Comment:
It should be now 1.0.1+30, but in general that works. Everytime we build new version, we need update pubspec.yaml with newer version number

Reply:
Thanks, that is fixed now.